### ISSUE TITLE
Why is this repo needed?

### ISSUE THREAD

**Issue by rlebeau** [flydev-fr/Indy10#1]
Just curious why this repo exists, since the [official Indy 10](https://github.com/IndySockets/Indy/) has support for Delphi 2007.

**Comment by flydev-fr** [comment]
Hi Remy,

From what I remember, I started coding with Delphi 2007 in 2016 and I started working on an upgrade one of our softwares. I then used Indy10.6 to get SSL working. Then I switched on Delphi Rio and after some years, I asked a colleague to upgrade his 2007 install that was containing the version 9.x of Indy. He had never used Github, so I uploaded the correct Indy folder that was working for the project with [instructions in FR](https://github.com/flydev-fr/Indy10/blob/master/UPGRADE%20TO%20INDY-10.6.txt) to upgrade his installation. This is the main reason.

Anyway, as we have our private repo with required deps as submodules, once I see you got my answer, I will delete it.
